home/forms/search.py

```diff
@@ -19,6 +20,17 @@ def get_domain_choices() -> list[Domain]:
     return choices
 
 
+def get_list_domain_choices() -> list[Domain]:
```

Comment:
Since nothing else is using `get_domain_choices` any more I think we can remove that implementation and replace it with this one.

I had a look at the subdomain choices and that uses a different function, `get_subdomain_choices`, so that should be unaffected. Not sure yet if we want to keep all that in the codebase but I'd prefer to wait and see until we have decided which domain model we're going to stick with.

Do you think it's worth wrapping the subdomains stuff in a feature flag so we skip any unnecessary calls?

Reply:
Awesome thanks Mat